lib/pingen/client.rb

```diff
@@ -30,6 +30,18 @@ def upload(pdf)
       post_multipart_request("/document/upload", pdf, {})
     end
 
+    def pdf(id)
+      get_request("/document/pdf/id/#{id}")
+    end
+
+    def delete(id)
+      post_request("/document/delete/id/#{id}", {})
+    end
+
+    def schedule_send(id, speed, color: false)
+      post_request("/document/send/id/#{id}", {color: color ? 1 : 0, speed: speed})
```

Comment:
what are the allowed values for `speed`?  maybe a comment would help to properly use it

Reply:
or maybe fast_send: true|false if it's about A or B post

Reply:
https://www.youtube.com/watch?v=bmFwnxdaEWw

Reply:
there could be more options but they also depend on the country, so it may get complicated. I'll go for `fast_send` and support only A or B post for now